JWPlayer_iOS_SDK.framework/SDKRouter.js

```diff
@@ -0,0 +1,57 @@
+var SDKRouter = {
```

Comment:
@henryhlee is there a way to make this file private ?

Reply:
When I was trying to make it inline, somehow it was not working at that time, so I didn't try to change it. However we should correct this in 3.5.1.